src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -252,4 +252,10 @@
   <data name="PX1032Title" xml:space="preserve">
     <value>DAC properties cannot contain method invocations</value>
   </data>
+  <data name="PX1030Fix" xml:space="preserve">
+    <value>Replace with the PXUnboundDefaultAttribute or set PersistingCheck to Nothing</value>
```

Comment:
@k-s-popova , please check this string.

Reply:
@saloponov, change the message as follows: 
Replace with PXUnboundDefaultAttribute or set PersistingCheck to Nothing